lib/task_bridge.rb

```diff
@@ -32,7 +32,7 @@ def initialize
       conflicts :personal_tags, :work_tags
       opt :services, "Services to sync tasks to", default: ENV.fetch("SYNC_SERVICES", "GoogleTasks,Github").split(",")
       opt :list, "Task list name to sync to", default: ENV.fetch("GOOGLE_TASKS_LIST", "My Tasks")
-      opt :repositories, "Github repositories to sync from", default: ENV.fetch("GITHUB_REPOSITORIES", []).split(",")
+      opt :repositories, "Github repositories to sync from", default: ENV.fetch("GITHUB_REPOSITORIES", "").split(",")
```

Comment:
I think the downstream error can be fixed by adding the `type` attribute (see https://github.com/ManageIQ/optimist/wiki/Option-Types):
```suggestion
      opt :repositories, "Github repositories to sync from", type: :strings, default: ENV.fetch("GITHUB_REPOSITORIES", "").split(",")
```
Which I was using at one point for other options and removed for some reason that I can't remember. But I think it should be fine here.